tools/validators/instance_validator/validate/telemetry_validator.py

```diff
@@ -22,12 +22,20 @@
 
 from validate import telemetry
 from validate import telemetry_error
+from validate import telemetry_warning
 
 DEVICE_ID = 'deviceId'
 
 class TelemetryValidator(object):
   """Validates telemetry messages against a building config file.
 
+  Attributes;
+    entities: a dict with entity_name as a key and EntityIntance as value.
+    timeout: the max time the validator must read messages from pubsub.
+    callback: the method called by the pubsub listener upon receiving a msg.
+    validated_entities: list of entities validated during the timeout duration.
```

Comment:
These two (validated_entities, validation_errors) should be private. I think that's indicated by prefixing them with an underscore and not mentioning them in the attributes doc?

Reply:
added _ and removed from attributes